psydac/feec/multipatch/operators.py

```diff
@@ -116,32 +130,72 @@ def __init__(self, V0h, domain_h, homogeneous_bc=False, kappa=1e10, tol=1e-12, v
         ah = discretize(a, domain_h, [V0h, V0h])    # ... or (V0h, V0h)?
 
         self._A = ah.assemble() #.toarray()
```

Comment:
I think that lines 115 to 132 should only be processed if `explicit == False`, correct?